lib/dotcom/utils/service_date_time.ex

```diff
@@ -0,0 +1,213 @@
+defmodule Dotcom.Utils.ServiceDateTime do
+  @moduledoc """
+  A collection of functions that helps to work with date_times with regard to service times.
+  Currently, we consider the most general case where service starts at 03:00:00am and ends at 02:59:59am.
+
+  In the future, we aim to add route-specific service times.
+
+  Note that functions like `service_today?/1` and `service_this_week?/1` are not exlusive.
+  A date_time can have service today and service this week.
+  I.e., today is both today and this week.
+
+  However, the `service_range/1` function *is* exclusive.
+  Given a date_time in today's service range, it will return :today.
+  """
+
+  require Logger
+
+  use Timex
+
+  import Dotcom.Utils.DateTime,
+    only: [coerce_ambiguous_time: 1, in_range?: 2, now: 0, timezone: 0]
+
+  alias Dotcom.Utils
+
+  @type service_range() :: :past | :today | :this_week | :next_week | :later
```

Comment:
(Non-blocking) This name (and the name of the function below that's also called `service_range/1`) feels like a potential source of confusion, since it refers to an atom, not an actual range.